### DISCUSSION TITLE
Unable to identify active tabs

### DISCUSSION THREAD

**Discussion by pete10-1-001** [sumatrapdfreader/sumatrapdf#3093] [General]
Hello
While I love the size of sumatrapdf, there is one major downside. I am not able to identify the active tab. Most of the time i have a lot of tabs open and its nearly unpossible to say which tab ist active. And as if this isn't enough, sumatrapdf doesen't show the full name of a pdf while the mouse is over a tab. Maybe that might not matter if you have just one tab opened, for me sumatrapdf is useless.
I can't understand at all why this isn't the case for anyone else. And for the developer himself.
Please fix this problem.
Pete

**Comment by GitHubRulesOK** [comment]
Hi-Light tab is white other is greyed out full name of web downloading file is visible

![image](https://user-images.githubusercontent.com/12754913/201543807-f72aaf25-d41f-4d04-affe-4233fda15b65.png)

**Comment by pete10-1-001** [comment]
Thanks for the answer.
Ok, I was on an older version, which was not able to diplay the full name of the pdf when the mouse is over a tab. Apart from that the problem not being able to identify the active tab is still there. Have a look at the picture of some opened pdfs. Can you quickly tell me the active pdf? I am not able to. Unfortunately.

![sumatra](https://user-images.githubusercontent.com/118131572/201653395-bd43cec6-0a9e-40af-8e92-f31f6e7226e5.png)

**Comment by tactilis** [comment]
> Can you quickly tell me the active pdf? I am not able to. Unfortunately.

It looks very clear to me on my laptop.

![image](https://user-images.githubusercontent.com/4060092/201658335-aa850919-3fa5-4f13-963e-86435539b7fb.png)
 
![image](https://user-images.githubusercontent.com/4060092/201659356-a5371fa3-4ca1-4c02-8c00-95c3e9ddd6ed.png)


If the lighter colour of the active tab does not stand out clearly on your laptop or monitor, then you might wish to check your brightness/contrast settings in case that is minimising the difference between active & background tabs.

**Comment by pete10-1-001** [comment]
I like sumatrapdf, therefore I use it on more than one machine. I can't believe that all these machines have monitors with wrong settings. And I will certainly not adjust all these monitors just for one program just because the developer thinks the difference between active and inactive tab is big enough. If nobody else has a problem with it it's ok but for me I won't use sumatrapdf any more. This is ridiculous.

**Comment by tactilis** [comment]
For comparison, here's the tab bar of SumatraPDF (at the top) and Acrobat (at the bottom).

Click on the image to see it full sized.

![image](https://user-images.githubusercontent.com/4060092/201693160-24ec9a94-53a1-4bff-8941-782e5e994c99.png)
 
The active tabs are the same colour.

The inactive tabs are very nearly the same colour.

If you look closely at the right hand Sumatra tab, you can see a little rectangle of very slightly darker grey (between the letters .pdf and the close tab icon) where I have pasted a piece of the Acrobat tab over the Sumatra tab. Acrobat is darker, but only just.

![image](https://user-images.githubusercontent.com/4060092/201694549-f3e71864-cacb-4b46-bb2e-3e6657483372.png)
 
I think the main difference is that the Sumatra tab bar is not as tall.

But honestly, as I said previously, the active tab is easy to spot on the laptop I am using.

**Comment by pete10-1-001** [comment]
Although the colors appear to be similar in SumatraPDF and Acrobat, the active tab in Acrobat is easieer to identify. You already mentioned it, the tabs are taller, despite of that the active tab in Acrobat has the "x" for closing the tab and the design of Acrobat seems to make it easier to recognize the active tab.

I did some searching and I found that the problem already mentioned in 2016. And it still exists. I am not the only one.
Either the developer ignores it or it is too hard to program (I don't think so) or there are too less people using sumatrapdf with more than one tab. I can't think of anything else. Or maybe it's me who's wrong.

**Reply by GitHubRulesOK** [reply]
you can change the colours in the source code see https://github.com/sumatrapdfreader/sumatrapdf/issues/1985 but the outer VisStudio themes control was designed for prior to windows 8 at which point many of the newer metro colours looked "flatter" in lighter or darker win 10 for reference this was one example  discussed in https://github.com/sumatrapdfreader/sumatrapdf/issues/743
![105420483-5a6dc780-5c38-11eb-8f28-c9f97d84ef40 1](https://user-images.githubusercontent.com/12754913/201702514-d9ef059f-5fe7-44aa-8089-c633c947df7f.png)

**Comment by nmackie** [comment]
I have to agree with Pete-10 here about the active tab not standing out. 
Using the light theme as an example, most of the top of the window is white, in roughly 3 rows. A border in the 1st/top row. A 2nd row with menu icon, Home tab and tabs of open files below it. And a row of icons/ tools/ status as the 3rd/bottom row. In the second row the active tab (often the most recently opened tab appearing on the right) is white and the inactive tabs are grey. Thus it is the inactive tabs that stand out and are bordered in white. The active tab does not stand out and has no visible borders, as it has the same background colour as the rows above and below.
<img width="885" height="73" alt="sumatrapdf" src="https://github.com/user-attachments/assets/bb940f1d-0f60-4151-a7f3-42f84176375d" />

**Comment by shimscharf** [comment]
I too find this a little difficult in my mind.
Using a any dark theme, the active tab is darker which is the opposite of what other programs do with tabs and hence is confusing
